### DISCUSSION TITLE
`graph publish` should update subgraph metadata if changed

### DISCUSSION THREAD

**Discussion by YaroShkvorets** [graphprotocol/graph-tooling#1707] [Ideas]
Right now, subgraph metadata is uploaded to IPFS and published on chain only for new subgraphs.

So if you are upgrading your subgraph to a new version, there is no way to change name/description/website, etc

**Comment by YaroShkvorets** [comment]
1. Upload json to IPFS
2. Execute `updateSubgraphMetadata` on L2GNS

**Comment by saihaj** [comment]
created https://github.com/graphprotocol/graph-tooling/discussions/1707
